src/components/SettingsDialog/SettingsDialog.vue

```diff
@@ -18,18 +18,37 @@
 		</NcAppSettingsSection>
 
 		<NcAppSettingsSection id="devices"
-			:name="t('spreed', 'Choose devices')"
+			:name="t('spreed', 'Devices')"
 			class="app-settings-section">
-			<MediaDevicesPreview />
+			<NcButton variant="secondary"
+				@click="openMediaSettings">
+				<template #icon>
+					<IconMicrophone :size="20" />
+				</template>
+				{{ t('spreed', 'Check devices') }}
+			</NcButton>
 			<NcCheckboxRadioSwitch v-if="supportStartWithoutMedia"
-				id="call-media"
 				:model-value="startWithoutMediaEnabled"
 				:disabled="mediaLoading"
 				type="switch"
-				class="checkbox call-media"
 				@update:model-value="toggleStartWithoutMedia">
 				{{ t('spreed', 'Turn off camera and microphone by default when joining a call') }}
 			</NcCheckboxRadioSwitch>
+			<NcCheckboxRadioSwitch v-if="supportDefaultBlurVirtualBackground"
+				type="switch"
+				:model-value="settingsStore.blurVirtualBackgroundEnabled"
+				@update:model-value="setBlurVirtualBackgroundEnabled">
+				{{ t('spreed', 'Enable blur background by default for all conversation') }}
```

Comment:
Typo in initial string 👀 
```suggestion
				{{ t('spreed', 'Enable blur background by default for all conversations') }}
```